src/main/webapp/app/shared/metis/posting-reactions-bar/posting-reactions-bar.component.ts

```diff
@@ -113,6 +113,7 @@ export class PostingReactionsBarComponent<T extends Posting> implements OnInit,
     canPin = false;
     channels: ChannelDTO[] = [];
     users: UserPublicInfoDTO[] = [];
+    originalAnswerId: number | undefined;
```

Comment:
you can use this here:
```suggestion
    originalAnswerId?: number;
```
does it even need to be undefined here?